framework/components/ecl-forms/ecl-forms-fieldsets/test/spec/fieldsets.js

```diff
@@ -0,0 +1,29 @@
+describe('forms-fieldset', () => {
```

Comment:
Completely agree. I'm not even sure it should be displayed at all. There's a need to refactor the docs here. If you have any idea, I'll appreciate :)